packages/extlib-compat/extlib-compat.1.6.1/opam

```diff
@@ -1,13 +1,17 @@
-opam-version: "1"
+opam-version: "1.2"
 maintainer: "ygrek@autistici.org"
+author: "ygrek@autistici.org"
```

Comment:
Here is how ygrek credited itself a few days ago: https://github.com/ocaml/opam-repository/pull/4302/files